app/css/app.css

```diff
@@ -86,81 +54,50 @@ button {
   border-radius: .3rem;
 }
 
-/* Modal */
-.modal {
```

Comment:
Since ```modal``` component is not used, I've just moved it's styles to a separate file (I think later one we'll just add build step to concatenate some files for example) and didn't do any changes.